.editorconfig

```diff
@@ -0,0 +1,17 @@
+# Editor configuration, see https://editorconfig.org
```

Comment:
Love this

(Though I think my editor doesn't use it yet)

Reply:
Which editor? The linked site has a very large list of editors that support it either directly or with a plugin.

Reply:
Besides VS Code, I've been using Helix recently. https://helix-editor.com/

Its a modal terminal based editor like Vim, but requires much less configuration (has a built in fuzzy finder, a discoverable UI, and build-in LSP support) to get up and running. My config is like 8 lines as opposed to hundreds of lines in my Neovim config.

Highly recommended! I think they have an issue open to support editor config.

Reply:
Interesting! I'm currently satisfied with PyCharm and Vim, but I'll still look into it.